src/reprostim-capture/videocapture/config.yaml

```
@@ -42,6 +43,28 @@ conduct_opts:
     # specify con/duct tool path/etc
     duct_bin: "duct"
 
+#
+# External process/hook to control Magewell USB capture device connect/disconnect events.
+# Can be used to monitor certain display connection status with "xrandr"
+# and automatically start/stop related subprocess/script.
+#
+ext_proc_opts:
+    # to enable external processes set "enabled" to "true"
```

in principle it could indeed be a list of multiple processes, but for now it is a single process, so 

```suggestion
    # to enable external process set "enabled" to "true"
```